app/experimenter/experiments/tests/test_tasks.py

```diff
@@ -532,20 +603,23 @@ def test_send_experiment_ending_email(self):
             proposed_start_date=date.today(),
             proposed_enrollment=0,
             proposed_duration=5,
+            is_paused=True,
```

Comment:
Had to add these so, the pause enrollment workflow isn't tested in these tests for now...will fix when we break down the update_experiment_info task?